src/routes/requestRoute.js

```diff
@@ -0,0 +1,126 @@
+const controller = require('../controllers/requestController');
+const { Router } = require('express');
+import authRequest from "../middlewares/check-auth";
+
+const router = Router();
+
+const createR = controller.createRequest;
+const updateR = controller.updateRequest; 
+const deleteR = controller.deleteRequest;
+const getR = controller.getAllRequests;
```

Comment:
You could refactor here by destructuring individual controllers from controller, eg, 
`const {
createRequest,
updateRequest,
deleteRequest,
getAllRequests
} = controller`
Then use createRequest instead of createR, etc. But you may leave this as is for now, in the interest of time